wiris/html-integrations
# Fix CKEditor 5 "modelWriter.createElement() is not a function" error

## Description

This pull request fixes issue #112, which throws an error when writing after a MathType formula in CKEditor 5.

## How to reproduce

1. Open a CKEditor 5 HTML5 demo
2. Write some text, followed by a space
3. Add a MathType equation, followed by a space
4. Type some text

Expected: the text is inserted, no error is thrown.
Actual: the text appears to be inserted, but an error is thrown, and the CKEditor engine doesn't actually process the inserted text.

For more details see #112.
